src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -56,5 +143,24 @@ private void ExportImplicitStyles(bool value)
 			// but, we can add them through merged-dict instead.
 			this.MergedDictionaries.Add(implicitResources);
 		}
+
+		private Style FindStyle(string key)
```

Comment:
By convention, Find can return null, Get must return a value (and throws if none is found).

```suggestion
		private Style GetStyle(string key)
```

Reply:
That's nice to know.